ui/src/message_popup/scaffolding_card.jsx

```diff
@@ -99,7 +119,7 @@ export default React.createClass({
   },
   
   render(){
-    const {itemsToShow} = this.props;
+    const {itemsToShow, allowChoosingResponseMode} = this.props;
```

Comment:
I wonder if we should make it so that `allowChoosingResponseMode` is enabled when the query string 'all' is present as well.